blocks/donor-wall/class-give-donor-wall.php

```diff
@@ -228,7 +232,8 @@ public function render_block( $attributes ) {
             'toggle_options'    => $attributes['toggleOptions'],
             'filter'            => $attributes['filter'],
 			'avatar_size'       => $avatarSize ?: 75,
-            'color'             => $attributes['color']
+            'color'             => $attributes['color'],
+            'show_timestamp'         => $attributes['showTimestamp'],
```

Comment:
@JoshuaHungDinh Can you fix this formatting issue

Reply:
@ravinderk will do, thanks for your help with these!

Reply:
Resolved by: https://github.com/impress-org/givewp/pull/6468/commits/3ae480d7ed6b079838668a6707efc27542893382